### PR TITLE
Fixed stop toggle

### PR DESCRIPTION
**Pull request by akgupta89**
https://github.com/trynmaps/opentransit-map/issues/90

Resolved this.

ESLint prevents me from setting the toggle to initialize on by default with a boolean of true, so I used 1 as an alternative to get around it.

### CODE REVIEW

File: src/ControlPanel.jsx [hunk 1]
```diff
@@ -81,8 +81,8 @@ class ControlPanel extends Component {
         <div className="routes-header stops-toggle">
           <h3>Stops</h3>
           <Toggle
-            defaultChecked={this.state.showStops}
-            onChange={() => this.setState({ showStops: !this.state.showStops })}
+            defaultChecked={1}
```
Comment (EddyIonescu) [inline src/ControlPanel.jsx]: would this work if we kept it with `this.state.showStops` (which is set to true)

File: src/ControlPanel.jsx [hunk 2]
```diff
@@ -106,6 +106,7 @@ class ControlPanel extends Component {
 ControlPanel.propTypes = {
   relay: propTypes.element.isRequired,
   filter: propTypes.element.isRequired,
+  stop: propTypes.element.isRequired,
```
Comment (EddyIonescu) [inline src/ControlPanel.jsx]: Could we make these names more intuitive? Could we leave them to `toggleStops`, `filterStops`?

File: src/ControlPanel.jsx [hunk 3]
```diff
@@ -106,6 +106,7 @@ class ControlPanel extends Component {
 ControlPanel.propTypes = {
   relay: propTypes.element.isRequired,
```
Comment (EddyIonescu) [inline src/ControlPanel.jsx]: just wondering, where is this defined?

File: src/ControlPanel.jsx (hunk 1 above)
Comment (akgupta89) [reply src/ControlPanel.jsx]: no, `this.state.showStops` isn't stored in the control panel component, it is in the parent Map component

File: src/ControlPanel.jsx (hunk 2 above)
Comment (akgupta89) [reply src/ControlPanel.jsx]: Good point, i'll change it

File: src/ControlPanel.jsx (hunk 3 above)
Comment (akgupta89) [reply src/ControlPanel.jsx]: I'm not sure, I copied it when I refactored Map and ControlPanel into separate files

File: src/ControlPanel.jsx (hunk 1 above)
Comment (EddyIonescu) [reply src/ControlPanel.jsx]:
```suggestion
             defaultChecked
```

**Pull request merged**